maestro-app/src/main/resources/config/application.yml

```diff
@@ -19,6 +19,9 @@ maestro:
       fileCentric:
           name: file_centric_1.0
           alias: file_centric
+      analysisCentric:
```

Comment:
we need to add an `enabled` property here to enable or disable a specific index, for both fileCentric and analysis centric 
so that if someone doesn't need one of them they can disable it